Updated Authentication section to reflect AWX only method. (#15602)
ansible/awx: awxkit/awxkit/cli/docs/source/authentication.rst

Authentication
==============

> The only way to authenticate AWX is with username and password by specifying them on every invocation as shown in the following example:

Reviewer: ```suggestion
To authenticate to AWX, include your username and password in each command invocation as shown in the following examples:
```
